src/containers/Login.vue

```diff
@@ -179,7 +128,6 @@ import Vue from 'vue';
 import firebase from 'firebase/app';
 
 import CustomFooter from '@/components/Footer.vue';
-import TopBar from '@/components/TopBar.vue';
```

Comment:
Now that the landing page does not use the topbar component with the CoursePlan logo, do we still want it on the 404 page? @tcho6319